_data/quality-prediction.yml

```diff
@@ -42,3 +42,15 @@
   urls:
     - https://omniscien.com/products/language-studio/
     - https://languagestudio.attachments5.freshdesk.com/data/helpdesk/attachments/production/12038641906/original/LSRestAPI_v5_Reference_Document_v1.0_20180531.pdf?response-content-type=application%2Fpdf&Expires=1662038771&Signature=QrS5lsOBmkFloMMh70jqVO9Qud2IdwLyPI7AOLClr5GLVXjNdYz1k9CXNGoQbVtdflRJU42E5YvqMeZyTSV36VRhrKUUFvYFLPouJwt98hZ7p0MLmbqgCyONa2Agc3GNHBavYYRYPgfwzEi3D1sMZxuhgLqg-Fs82o-~hj~2dA~77aUNc0hGpTrUN5flpfSg1AYcHP7GreZiCrAehfBHmZRnnGrU7B8EQLfDKrCZKi6YB2JxpuJEFuXp8ynkE85wXx6FEW0svsqsXfsEUHXWAsAGR0tKkJ0r2ndkoJiOK0hroU3MWPY4Yc28eFKAbnTNfrbNetx0M1QCV411n3IWug__&Key-Pair-Id=APKAJ7JARUX3F6RQIXLA ]
+
+- id: google
+  name: MTQP
+  description: Machine translation quality prediction scores
+  languages: [ ['en', 'ar'],  ['ar', 'en'],  ['en', 'zh'],  ['zh', 'en'],  ['en', 'fr'],  ['fr', 'en'],  ['en', 'de'],  ['de', 'en'],  ['en', 'hi'],  ['hi', 'en'],  ['en', 'it'],  ['it', 'en'],  ['en', 'ja'],  ['ja', 'en'],  ['en', 'pt'],  ['pt', 'en'],  ['en', 'ru'],  ['ru', 'en'],  ['en', 'es'],  ['es', 'en'] ]
```

Comment:
The rest of the engines' QE language pairs are not spelled within single quotes, shall we change that for consistency?

```
languages: [ [ 'en', 'ar' ] ]
```

versus

```
languages: [ [ en, ar ] ]
```

Reply:
I'll rm the quotes.